### PR TITLE
Update construct_single_presto_input to unmask NDVI if its calculated

### PR DESCRIPTION
**Pull request by gabrieltseng**
Before, NDVI would always be masked in `construct_single_presto_input`. However, if `B4` and `B8` band values are passed NDVI can be calculated by `S1_S2_ERA5_SRTM.normalize`. 

This PR unmasks NDVI if its calculated by `S1_S2_ERA5_SRTM.normalize`.

### CODE REVIEW

File: presto/dataops/utils.py [hunk 1]
```diff
@@ -60,8 +61,14 @@ def construct_single_presto_input(
     if dynamic_world is None:
         dynamic_world = torch.ones(num_timesteps) * (DynamicWorld2020_2021.class_amount)
 
+    keep_indices = [idx for idx, val in enumerate(BANDS) if val != "B9"]
+    mask = mask[:, keep_indices]
+
     if normalize:
-        keep_indices = [idx for idx, val in enumerate(BANDS) if val != "B9"]
-        mask = mask[:, keep_indices]
         x = S1_S2_ERA5_SRTM.normalize(x)
```
Comment (rubencart) [inline presto/dataops/utils.py:68]: Can you add a comment here that `.normalize` includes `x = x[:, keep_indices]`? :)

Comment (gabrieltseng) [reply presto/dataops/utils.py:69]: added!
